### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by vbevziuk**
- [DEMO LINK](https://vbevziuk.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://vbevziuk.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,53 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+        src="images/logo.png"
+        alt="Moyo"
+        class="logo__img">
+      </a>
```
Comment (etojeDenys) [inline src/index.html:21]:
```suggestion
     <img
        src="images/logo.png"
        alt="Moyo"
        class="logo__img"
     >
      </a>
```

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,53 @@
...
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item" >
+            <a
+              href="index.html"
+              class="nav__link"
+              data-qa="hover"
+            >Laptops & Computers</a>
```
Comment (etojeDenys) [inline src/index.html:38]: image

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,60 @@
...
+  line-height: 60px;
+  margin-left: 20px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
```
Comment (etojeDenys) [inline src/style.css:46]: move to some var and reuse everywhere

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,60 @@
...
+}
+
+.nav__link {
+  position: relative;
+  display: block;
+  text-decoration: none;
+  color: #000;
+  height: 60px;
+  line-height: 60px;
+  margin-left: 20px;
```
Comment (etojeDenys) [inline src/style.css:38]: [let's organize your CSS](https://9elements.com/css-rule-order/)

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 5]
```diff
@@ -7,8 +7,55 @@
...
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Video</a>
+          </li>
```
Comment (VitaliyBodnar) [inline src/index.html:53]: please add the spaces between the li blocks, as it will make a code cleaner

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
Almost done!

File: src/index.html [hunk 6]
```diff
@@ -7,8 +7,62 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo"
+          class="logo__img">
```
Comment (polosanya) [inline src/index.html:20]:
```suggestion
        <img
          src="images/logo.png"
          alt="Moyo"
          class="logo__img"
        >
```

File: src/index.html [hunk 7]
```diff
@@ -7,8 +7,62 @@
...
+          </li>
+
+          <li class="nav__item" >
+            <a
+              href="index.html"
+              class="nav__link"
+              data-qa="hover"
+            >
+            Laptops & Computers
+            </a>
```
Comment (polosanya) [inline src/index.html:43]:
```suggestion
            >
              Laptops & Computers
            </a>
```

**Review by polosanya — APPROVED** [review]
Good job! I'm approving it, but please pay attention to the comments)
The title is redundant here. You can remove it
image

File: src/index.html (hunk 6 above)
Comment (vbevziuk) [reply src/index.html:20]: the thing is that I made the same changes, but for some reason they were not saved

**Pull request closed without merging**